datalad/interface/tests/test_save.py

```diff
@@ -363,4 +366,48 @@ def test_symlinked_relpath(path):
         ds.repo.add(later, git=True)
         ds.save("committing", path=later)
 
-    known_failure_v6(ok_clean_git)(dspath)
\ No newline at end of file
+    known_failure_v6(ok_clean_git)(dspath)
+
+
+def test_bf1886(path):
```

Comment:
`@with_tempfile(mkdir=True)` is missing?  causes
```
======================================================================
ERROR: datalad.interface.tests.test_save.test_bf1886
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
TypeError: test_bf1886() takes exactly 1 argument (0 given)
```

Reply:
Conflict resolution gone bad...